src/index.html

```diff
@@ -7,6 +7,40 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar">
+      <ul class="calendar__container calendar__container--month-length--29">
+        <li class="calendar__day calendar__day--1 calendar__day--start-day--sun"></li>
```

Comment:
the modifier of length and first day should be in the one bock, let's move `calendar__day--start-day--sun` to `<ul>` and replace `calendar__day--1` with css `:first-child`